python_testcontainers/infrahub_testcontainers/docker-compose.test.yml

```diff
@@ -82,10 +98,13 @@ services:
       retries: 5
 
   infrahub-server:
+    deploy:
+      mode: replicated
+      replicas: ${INFRAHUB_TESTING_WEB_CONCURRENCY}
     image: "${INFRAHUB_TESTING_DOCKER_IMAGE}:${INFRAHUB_TESTING_IMAGE_VERSION}"
     command: >
       gunicorn --config backend/infrahub/serve/gunicorn_config.py
-      -w ${INFRAHUB_TESTING_WEB_CONCURRENCY}
+      -w 1
```

Comment:
to be as close as possible to a real customer environment, I think we should keep 4 gunicorns worker for now and maybe start with 2 `INFRAHUB_TESTING_WEB_CONCURRENCY` 

I'm happy to revisit that later but if we change our recommendations we need to ensure that customers in production are running a similar setup

Reply:
We should change our recommendations when using a load balancer then.
Having 4 gunicorns by default seems a bit overkill to me (it would lead to too much memory usage). Having less gunicorn workers would let the container orchestrator schedule the infrahub-server containers better, thus distributing memory usage.

I have reverted that change and introduced another environment variable for the API server replica count.

Reply:
Let's revisit that once we have some real data to measure the impact of this change :)